lizard_auth_client/conf.py

```diff
@@ -21,6 +21,7 @@ class SSOAppConf(AppConf):
     CREDENTIAL_CACHE_TIMEOUT_SECONDS = 60
     JWT_EXPIRATION_MINUTES = 5
     JWT_ALGORITHM = 'HS256'
+    SERVER_JWT_ALGORITHMS = ["HS256"]
```

Comment:
Ik neem aan dat het klopt (enkelvoud/meervoud)?

Als alles gemerged is moeten we er ook maar eens 'black' overheen gooien. Ik vind die single quotes er niet meer uitzien :-)